### PR TITLE
Create a new script which used to get missed e2e profiles in upgrade

### PR DESCRIPTION
**Pull request by JianLi-RH**
/cc @jianlinliu @evakhoni @shellyyang1989 @jiajliu @dis016 @liangxia 

Verified it on my local machine:
```
[jianl@jianl-thinkpadt14gen4 release]$ e2e_yaml="/home/jianl/1_code/release/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml"
upgrade_yaml="/home/jianl/1_code/release/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml"
[jianl@jianl-thinkpadt14gen4 release]$ python ci-operator/config/openshift/openshift-tests-private/tools/get_missed_jobs.py $e2e_yaml $upgrade_yaml
- as: aws-c2s-ipi-disc-priv-fips-f2
- as: aws-ipi-disc-priv-mtu-localzone-fips-f2-nokubeadmin
- as: aws-ipi-disc-priv-workers-rhcos-rhel8-f14
- as: aws-ipi-efa-pg-f5
- as: aws-ipi-localzone-byo-subnet-ovn-day2-f2
- as: aws-ipi-localzone-byo-subnet-ovn-f2
- as: aws-ipi-localzone-fips-f2
- as: aws-ipi-outpost-day2-f2
- as: aws-ipi-ovn-efs-fips-f14-cert-manager-custom-api-ingress
- as: aws-ipi-ovn-ipsec-f2-obo
- as: aws-ipi-ovn-ipsec-to-multiarch-f2
- as: aws-ipi-ovn-winc-f2
- as: aws-ipi-private-shared-vpc-phz-sts-f2
- as: aws-ipi-proxy-fips-f14-fips-check
- as: aws-ipi-proxy-sdn-fips-f14
- as: aws-ipi-shared-vpc-phz-freeipa-f14
- as: aws-ipi-shared-vpc-phz-sts-openldap-fips-f2
- as: aws-ipi-wavelength-byo-subnet-ovn-day2-f5
- as: aws-ipi-wavelength-byo-subnet-ovn-f2
- as: aws-ipi-wavelength-byo-subnet-ovn-mtu-day2-f5
- as: aws-ipi-wavelength-fips-f2
- as: aws-ipi-workers-marketplace-f5
- as: aws-sc2s-ipi-disc-priv-fips-f2
- as: aws-usgov-ipi-disc-priv-f2
- as: aws-usgov-ipi-private-ep-fips-f5
- as: aws-usgov-ipi-private-f5
- as: aws-usgov-ipi-private-sts-f5
- as: aws-usgov-ipi-private-workers-marketplace-f5
- as: azure-ipi-confidential-trustedlaunch-f5
- as: azure-ipi-marketplace-mini-perm-f5
- as: azure-ipi-proxy-workers-rhcos-rhel8-f14
- as: azure-ipi-to-multiarch-f2
- as: azure-ipi-vmgenv1-f5
- as: azure-mag-ipi-fips-f2
- as: azure-mag-ipi-fullyprivate-f2
- as: azure-stack-ipi-f28
- as: azure-stack-ipi-proxy-fips-f28
- as: azure-stack-upi-f28
- as: baremetal-3-plus-1-agent-ipv4-static-f14
- as: baremetal-compact-agent-platform-none-ipv4-static-f14
- as: baremetal-ha-agent-ipv4-static-disc-fips-f7
- as: baremetal-ha-agent-platform-none-ipv4-static-f14
- as: baremetal-ipi-ovn-lvms-f14
- as: baremetal-pxe-compact-agent-ipv4-static-connected-f14
- as: baremetal-pxe-ha-agent-ipv4-static-connected-f14
- as: baremetal-pxe-sno-agent-ipv4-static-connected-f14
- as: baremetal-sno-agent-ipv4-static-connected-fips-f7
- as: baremetal-upi-ovn-ipv4-f14
- as: baremetalds-ipi-ovn-dualstack-primaryv6-f7
- as: baremetalds-ipi-ovn-ipv4-fips-f14
- as: baremetalds-ipi-ovn-ipv4-fips-f14-wrs
- as: baremetalds-ipi-ovn-ipv6-fips-f14
- as: gcp-ipi-marketplace-f28
- as: gcp-ipi-mini-perm-custom-type-f28
- as: gcp-ipi-oidc-rt-fips-regen-cert-f14
- as: gcp-ipi-ovn-winc-f7
- as: gcp-ipi-proxy-etcd-encryption-f28-security-profiles-operator
- as: gcp-ipi-secureboot-confidential-computing-f14
- as: gcp-ipi-to-multiarch-f14
- as: gcp-ipi-workers-rhel8-fips-f28
- as: ibmcloud-ipi-dedicatedhost-dynamic-f28
- as: ibmcloud-ipi-disc-priv-f28
- as: ibmcloud-ipi-mini-perm-f28
- as: ibmcloud-ipi-ovn-ipsec-f28
- as: ibmcloud-ipi-ovn-mtu-migrate-f28
- as: ibmcloud-ipi-private-byo-kms-f28
- as: ibmcloud-ipi-private-custom-cos-endpoint-fips-f28
- as: ibmcloud-ipi-private-fips-f28
- as: ibmcloud-ipi-proxy-private-rt-f14
- as: ibmcloud-ipi-tp-f28
- as: nutanix-ipi-boot-categories-project-rt-f28
- as: nutanix-ipi-compact-f28
- as: nutanix-ipi-disc-rt-f28
- as: nutanix-ipi-image-registry-s3-f28
- as: nutanix-ipi-proxy-fips-mini-perm-f28
- as: nutanix-ipi-tp-f28
- as: nutanix-ipi-zones-external-lb-f28
- as: nutanix-ipi-zones-f14
- as: telcov10n-hub-spoke-gitops-ztp-f360
- as: vsphere-agent-compact-day2-add-nodes-f14
- as: vsphere-agent-compact-fips-f14
- as: vsphere-agent-disc-compact-day2-add-nodes-f14
- as: vsphere-agent-disc-compact-f14
- as: vsphere-agent-disc-ha-f14
- as: vsphere-agent-disc-sno-f14
- as: vsphere-agent-dualstack-compact-day2-add-nodes-f14
- as: vsphere-agent-dualstack-compact-f14
- as: vsphere-agent-dualstack-ha-f14
- as: vsphere-agent-dualstack-sno-f14
- as: vsphere-agent-ha-day2-add-nodes-f14
- as: vsphere-agent-ha-f14
- as: vsphere-agent-sno-day2-add-nodes-f14
- as: vsphere-agent-sno-f14
- as: vsphere-ipi-amd-f7-netobserv
- as: vsphere-ipi-compact-etcd-encryption-f28
- as: vsphere-ipi-disc-f28
- as: vsphere-ipi-disk-encryption-tang-fips-f28
- as: vsphere-ipi-external-lb-post-f28
- as: vsphere-ipi-multisubnets-elb-usermanaged-f28
- as: vsphere-ipi-ovn-dualstack-f28
- as: vsphere-ipi-ovn-dualstack-privmaryv6-f28
- as: vsphere-ipi-ovn-ipsec-mininal-permission-f28
- as: vsphere-ipi-ovn-local-to-shared-gateway-mode-migration-f14
- as: vsphere-ipi-ovn-shared-to-local-gw-migration-f28
- as: vsphere-ipi-proxy-fips-f28
- as: vsphere-ipi-proxy-tp-f28
- as: vsphere-ipi-remote-worker-f28
- as: vsphere-ipi-sdn-migration-ovn-f14
- as: vsphere-ipi-static-ip-f28
- as: vsphere-ipi-template-usertags-f28
- as: vsphere-ipi-transit-masq-subnet-f28
- as: vsphere-ipi-zones-f28
- as: vsphere-ipi-zones-multisubnets-external-lb-f28
- as: vsphere-upi-disc-rhcos-rhel8-f28
- as: vsphere-upi-disc-secureboot-fips-f28
- as: vsphere-upi-disconnecting-network-f28
- as: vsphere-upi-encrypt-f28
- as: vsphere-upi-platform-external-f28
- as: vsphere-upi-platform-none-f28
- as: vsphere-upi-zones-f28
[jianl@jianl-thinkpadt14gen4 release]$
```

### CODE REVIEW

**Comment by shellyyang1989** [comment]
@liangxia PTAL Thank you!

File: ci-operator/config/openshift/openshift-tests-private/tools/get_missed_jobs.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+"""
+This script is used to find out missed e2e profiles in upgrade config file.
+When we adding new upgrade config files, we can compare e2e config file with upgrade config file, then find out the jobs have not automated in upgrade.
+
+Here is the example to run this script:
+```
+e2e_yaml="/home/jianl/1_code/release/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml"
+upgrade_yaml="/home/jianl/1_code/release/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml"
+
+python ci-operator/config/openshift/openshift-tests-private/tools/get_missed_jobs.py $e2e_yaml $upgrade_yaml
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/tools/get_missed_jobs.py:10]: Perhaps rename `get_missed_jobs.py` to `get_missed_upgrade_jobs.py` based on its purpose ?

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/tools/get_missed_jobs.py]: done

**Comment by evakhoni** [comment]
nice one!

**Pull request merged**